### PR TITLE
Security vulnerability fixes

### PR DESCRIPTION
**Pull request by martinchacon**
- Low severity security vulnerability detected in debug <2.6.9 [1]
- High severity security vulnerability detected in lodash <4.17.5 [2]

[1] https://nvd.nist.gov/vuln/detail/CVE-2017-16137
[2] https://nvd.nist.gov/vuln/detail/CVE-2018-3721

### CODE REVIEW

**Comment by haruki-yamanashi** [comment]
Link for NVD report in commit message is not correct. Please fix.

**Comment by martinchacon** [comment]
Done.

File: package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
         "keystone": "./lib/keystone"
     },
     "dependencies"   : {
-        "lodash"      : "3.9.x",
-        "debug"       : "2.2.x",
+        "lodash"      : "4.17.x",
```
Comment (witekest) [inline package.json]: nit: should be `~4.17.5`

File: package.json [hunk 2]
```diff
@@ -27,8 +27,8 @@
...
+        "debug"       : "2.6.x",
```
Comment (witekest) [inline package.json]: `~2.6.9`

File: package.json (hunk 1 above)
Comment (martinchacon) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (martinchacon) [reply package.json]: Done

**Pull request merged**